tests/test_pytorch_data_teacher.py

```diff
@@ -56,6 +56,7 @@ def solved_task(str_output, valid, test):
     )
 
 
+@testing_utils.skipIfGPU
```

Comment:
is this broken on gpu?

Reply:
No, but it doesn't mix well with the multiprocessing_train. (Multiprocessing_train changes the spawn method which creates issues).